consumer-configuration/README.adoc

```diff
@@ -0,0 +1,347 @@
+
+//Additional line break to resolve mod docs generation error
+
+[id="con-adding-consumer-group-id_{context}"]
+== How to add a consumer to a consumer group
+
+A group ID is _required_ for a consumer to be able to join a consumer group.
+
+When you configure a consumer client application to access Kafka, you assign a group ID to associate it with a consumer group.
+All consumers with the same group ID belong to the consumer group with that ID.
+Consumer groups connected to a Kafka instance are presented in the {product} web console.
+
+The group ID is the _name_ of the consumer group.
+Add it as a property to your consumer configuration.
+The group ID can't be null or empty.
+
+.Example group ID property configuration for a consumer
+[source,properties]
+----
+group.id=my-group-id
+----
+
+.Property attributes
+[cols="25%,75%"]
+|===
+
+h|Type
+|String
+
+h|Default value
+|None
+
+h|Supported values
+|Letters (Aa-Zz), numbers (0-9), underscores ( _ ), or hyphens ( - )
+|===
+
+If you are using Kafka bin scripts to consume messages, you can use the `kafka-console-consumer.sh` tool to create a group ID:
+[source,subs="+quotes,+attributes"]
+----
+bin/kafka-console-consumer.sh --bootstrap-server __<bootstrap_server>__ --topic test-topic --group my-consumer-group
+----
+
+The `<bootstrap_server>` is the bootstrap server endpoint for your service account.
+You copy this information for the Kafka instance in the {product} web console by selecting the options menu (three vertical dots) and clicking *View connection information*.
+
+[NOTE]
+====
+Adding more consumers to a group than there are partitions will not help throughput, but it does mean that there are consumers on standby should one stop functioning.
+If you can meet throughput goals with fewer consumers, you save on resources.
+====
+
+.Additional resources
+* {base-url}{getting-started-url}[_Getting started with {product}_^]
+* https://kafka.apache.org/documentation/#consumerconfigs[Consumer Configs^] in Kafka documentation
+
+[id="proc-editing-consumer-group-properties_{context}"]
+== Reviewing consumer group properties in {product}
+
+[role="_abstract"]
+Use the {product} web console to select a consumer group and review the consumer properties.
+You can look at the consumer groups associated with a Kafka instance or a specific topic of the instance.
+
+As an alternative to using the {product} web console, you can use the `rhoas` command-line interface (CLI) to return information on a consumer group, as shown in the following example command:
+
+.Example CLI command to return consumer group information
+[source]
+----
+rhoas kafka consumergroup describe [flags]
+----
+
+For a list of consumer group commands that you can use with the CLI, see the {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^].
+
+If you are using Kafka bin scripts, you can use the `kafka-consumer-groups.sh` tool to provide the same information:
```

Comment:
Ditto, "you're using". Best to be consistent with contractions. Maybe search all instances of "you are" and any others you think and contract as needed for consistency?

Reply:
Updated :+1: